code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -121,7 +121,7 @@
 
 /obj/item/reagent_containers/spray/pepper
 	name = "pepperspray"
-	desc = "Manufactured by Uhang Inc., it fires a mist of condensed capsaicin to blind and down an opponent quickly."
```

Comment:
put the period back. Inc. is short for incorporated so the grammar is correct here actually.

Reply:
> put the period back. Inc. is short for incorporated so the grammar is correct here actually.

Thought that as well but thought it would be useless and proper quotation would be better instead.